pkcs11/gkm/test-sexp.c

```diff
@@ -101,6 +133,34 @@ test_parse_key (Test *test, gconstpointer unused)
 	g_assert (ret);
 	g_assert (mpi != NULL);
 	gcry_mpi_release (mpi);
+
+	/* DSA */
```

Comment:
that's definitely a good approach, but to show (and allow for easy verify) that you haven't broken anything, I think it's nicer when the test case for existing functionality is added before the code is modified  - you know, like in the PR you reviewed recently 😉